lxd/instance/drivers/load.go

```diff
@@ -29,6 +29,8 @@ func init() {
 
 	// Expose create to the instance package, to avoid circular imports.
 	instance.Create = create
+
+	InitIdmappedMountsMap()
```

Comment:
@brauner it would be good to keep this driver specific function call out of the driver-agnostic parts of the package. You could initialise the map directly from `lxcInstantiate`.

Reply:
Yeah, that's a good idea. I moved this whole thing into the storage driver part where it seems to fit more as this is really instance driver agnostic code.